src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/framework-routing-1.0.xsd

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<xsd:schema xmlns="http://symfony.com/schema/routing"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="http://symfony.com/schema/routing"
+            elementFormDefault="qualified">
+
+  <xsd:redefine schemaLocation="https://symfony.com/schema/routing/routing-1.0.xsd">
+    <xsd:complexType name="routes">
+      <xsd:complexContent>
+        <xsd:extension base="routes">
+          <xsd:choice minOccurs="0" maxOccurs="unbounded">
+            <xsd:element name="template-route" type="template-route" />
+            <xsd:element name="redirect-route" type="redirect-route" />
+            <xsd:element name="url-redirect-route" type="url-redirect-route" />
+            <xsd:element name="gone-route" type="gone-route" />
+          </xsd:choice>
+        </xsd:extension>
+      </xsd:complexContent>
+    </xsd:complexType>
+  </xsd:redefine>
+
+  <xsd:group name="base-configs">
+    <xsd:choice>
+      <xsd:element name="path" type="localized-path" />
+      <xsd:element name="requirement" type="element" />
+      <xsd:element name="option" type="element" />
+      <xsd:element name="condition" type="xsd:string" />
+    </xsd:choice>
+  </xsd:group>
+
+  <xsd:complexType name="base-route">
+    <xsd:attribute name="id" type="xsd:string" use="required" />
+    <xsd:attribute name="path" type="xsd:string" />
+    <xsd:attribute name="host" type="xsd:string" />
+    <xsd:attribute name="methods" type="xsd:string" />
+    <xsd:attribute name="locale" type="xsd:string" />
+    <xsd:attribute name="format" type="xsd:string" />
+    <xsd:attribute name="utf8" type="xsd:boolean" />
+  </xsd:complexType>
```

Comment:
I don't quite understand XSD - and how this relates to the original XSD file. But... do we need `methods` here?

Reply:
This is defining a group a base `elements` for the new route types, basically everything same as a classic route but without the `controller`. There should be no reason to remove support for one one them here IMHO as we do it consistently in other formats too (YAML and PHP-DSL), and also because it may cause some BC issues?